app/Http/Requests/AuthenticationRequest.php

```diff
@@ -13,15 +15,37 @@ class AuthenticationRequest extends FormRequest
      */
     public function rules(): array
     {
-        return [
-            'email' => [
-                'required',
-                'email:filter',
-                'unique:users,email',
-                'max:255',
-                'string'
-            ]
-        ];
+        if((int)$this->authentication_type === AuthenticationType::USER_REGISTER){
+            return [
+                'email' => [
+                    'required',
+                    'email:filter',
+                    'unique:users,email',
```

Comment:
```suggestion
                    'unique:users,email',
```
return ruleの前に変数に返却値を入れるようにするなどして、この内容のみ条件分岐するようにしてください。

Reply:
rules内の記述を修正致しまいた。ご確認よろしくお願い致します。
```
        $email_validation = match ((int)$this->authentication_type) {
            AuthenticationType::USER_REGISTER => 'unique:users,email',
            AuthenticationType::PASSWORD_RESET => 'exists:users,email',
            default => ''
        };
            return [
                'email' => [
                    'required',
                    'email:filter',
                    'max:255',
                    'string',
                    $email_validation
                ],
```
app/Http/Requests/AuthenticationRequest.php